### DISCUSSION TITLE
Build tried for 4 hours to start then failed, don't want to be billed

### DISCUSSION THREAD

**Discussion by paperview** [codemagic-ci-cd/codemagic-docs#1753] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

This is just a one off it seems, want to make sure we aren't charged seems like a bug.  Builds before and after work fine.
URL to build:
https://codemagic.io/app/60b15535c0045300126680a3/build/6485064d4b3f7553efd73785

### Expected results

For a typical build in this project it takes 15 minutes

### Actual results

Build unexpectedly lasted 4 hours.  
URL to build:
https://codemagic.io/app/60b15535c0045300126680a3/build/6485064d4b3f7553efd73785

### Build id (optional)

6485064d4b3f7553efd73785

**Comment by dtrdic** [comment]
hey @paperview, this issue should be fixed now, can you confirm it?
